src/vector_index.c

```diff
@@ -68,7 +68,33 @@ VecSimIndex *OpenVectorIndex(RedisSearchCtx *ctx,
   return openVectorKeysDict(ctx, keyName, 1);
 }
 
-IndexIterator *NewVectorIterator(RedisSearchCtx *ctx, VectorQuery *vq) {
+static bool isFit (VecSimIndex *ind, size_t size) {
```

Comment:
can we make it is a library service on a different PR?

Reply:
I think we should add a check for that in the library. Maybe when we add support for more types?